app/Http/Controllers/GroupController.php

```diff
@@ -23,6 +25,34 @@ public function __construct()
         $this->beforeFilter('auth', array('on' => array('post', 'put', 'delete')));
     }
 
+    /**
+     * Store newly created Group.
+     *
+     * @return Response
+     */
+    public function store(StoreGroupRequest $request)
```

Comment:
Nice.  I think Laravel's choice of conventions is dumb, but we will go with it.

![](http://i.imgur.com/z3NMd4q.gif)